src/test/e2e/swagger/swagger-ui-interactions.spec.ts

```
@@ -0,0 +1,261 @@
+  test('should handle endpoint parameter inputs', async ({ page }) => {
+    // Find an endpoint with parameters
+    const endpointWithParams: Locator = page.locator('.opblock').filter({ hasText: '{' }).first();
+    await endpointWithParams.click();
+
+    // Look for parameter inputs
+    const paramInputs: Locator = endpointWithParams.locator('input[type="text"]');
+    const paramCount: number = await paramInputs.count();
+
+    if (paramCount > 0) {
+      // Fill in parameters
+      for (let i: number = 0; i < paramCount; i += 1) {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)